src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -317,32 +317,28 @@ private static String createPropertiesText() {
 
             String typeText = "java.lang.String[]";
             final String propertyType = property.get(2).getTextContent();
-            final boolean isPropertyTokenType = propertyType.contains("subset of tokens")
+            final boolean isSpecialAllTokensType = propertyType.contains("all tokens");
```

Comment:
`set of any tokens` is more logical, sorry for confusion from the beginning.

Reply:
Done